network-store-client/src/main/java/com/powsybl/network/store/client/RestClient.java

```diff
@@ -24,16 +25,14 @@ public interface RestClient {
 
     <T extends IdentifiableAttributes> List<Resource<T>> getAll(String target, String url, Object... uriVariables);
 
-    <T extends IdentifiableAttributes> void update(String url, Resource<T> resource, Object... uriVariables);
+    <T extends Attributes> void updateAll(String url, List<Resource<T>> resources, Object... uriVariables);
```

Comment:
maybe here or in a separate PR, rename things so that we have more symmetry between create/updateAll (either create/update or createAll/updateAll)

Reply:
create renamed to createAll